docs/pages/inboxes/user-consent/support-user-consent.md

```diff
@@ -2,29 +2,29 @@
 
 Use the following methods to provide users with control over their messaging experience, ensuring their inboxes are tailored to their preferences and spam-free.
 
-## Get new consent records from the network
-
-Get the latest consent records from the network:
-
-:::tip[Note]
-Consent syncing is backed by XMTP's history system. Syncing works only if a [history sync URL](/inboxes/create-a-client/#configure-an-xmtp-client) is specified on client create. By default, the URL to an Ephemera-hosted history sync server is set.
-:::
-
-:::code-group
-
-```tsx [React Native]
-await client.conversations.syncAllConversations()
-```
-
-```kotlin [Kotlin]
-client.conversations.syncAllConversations()
-```
-
-```swift [Swift]
-try await client.conversations.syncAllConversations()
-```
-
-:::
+## Get new consent records from the network 
```

Comment:
Should we update this to use the independent sync group call instead? Or as an alternative?